### DISCUSSION TITLE
Adding new repos

### DISCUSSION THREAD

**Discussion by aaguiarz** [cncf/clomonitor#678] [Q&A]
Hi!

We added this repository https://github.com/openfga/python-sdk to the OpenFGA organization. How can we get it analyzed?

Thanks

**Comment by eddie-knight** [comment]
Hey @aaguiarz, the data files are used to track projects for scanning.

All it takes is to make a PR to [add the new entry](https://github.com/cncf/clomonitor/blob/main/data/cncf.yaml#L2361-L2393) for the repo to be tracked.
